apps/token/src/routes/governance/proposals/proposals-container.tsx

```diff
@@ -24,6 +24,7 @@ export const ProposalsContainer = () => {
   const { t } = useTranslation();
   const { data, loading, error } = useQuery<Proposals, never>(PROPOSALS_QUERY, {
     pollInterval: 5000,
+    errorPolicy: 'ignore',
```

Comment:
chore (non-blocking): Can we add a comment here to serve as a reminder that this is to get around some backend issues and should be removed in future.